src/test/java/uk/ac/ebi/eva/commons/models/converters/data/VariantSourceEntryToDBObjectConverterTest.java

```diff
@@ -63,6 +63,7 @@ public void setUp() {
         file.addAttribute("QUAL", "0.01");
         file.addAttribute("AN", "2");
         file.addAttribute("MAX.PROC", "2");
+        file.setFormat("gzip");
```

Comment:
Why this?

Reply:
This is so because of the new VariantSourceEntryMongo. The `samp` field doesn't populate if `format` field of `file` isn't set. So that the `samp` field doesn't populate, which results in following test assertion-error: 

VariantSourceEntryToDBObjectConverterTest.testConvertToStorageTypeWithSamples:114 expected:<{ "fid" : "f1" , "sid" : "s1" , "attrs" : { "QUAL" : "0.01" , "AN" : "2" , "MAX?PROC" : "2"} , "samp" : { "def" : "0/0" , "0/1" : [ 1] , "1/1" : [ 2]}}> but was:<{ "fid" : "f1" , "sid" : "s1" , "attrs" : { "QUAL" : "0.01" , "AN" : "2" , "MAX?PROC" : "2"}}>

Added the following line to correct this: 
```file.setFormat("gzip");```

which then allowed for `samp` to be set, but now also required that `mongoFileWithIds` has `format` field present, as you can see in following test assertion-error message:
```
VariantSourceEntryToDBObjectConverterTest.testConvertToStorageTypeWithSamples:115 expected:<{ "fid" : "f1" , "sid" : "s1" , "attrs" : { "QUAL" : "0.01" , "AN" : "2" , "MAX?PROC" : "2"} , "samp" : { "def" : "0/0" , "0/1" : [ 1] , "1/1" : [ 2]} , **"fm" : "gzip"**}> but was:<{ "fid" : "f1" , "sid" : "s1" , "attrs" : { "QUAL" : "0.01" , "AN" : "2" , "MAX?PROC" : "2"}}>
```
So also added `mongoFileWithIds.put("fm", "gzip");` line to `mongoFileWithIds` because now we have `file.setFormat("gzip");`.